src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -192,6 +192,15 @@
   <data name="DoNotDisableSchUseStrongCryptoMessage" xml:space="preserve">
     <value>{0} disables TLS 1.2 and enables SSLv3</value>
   </data>
+  <data name="DoNotHardCodeEncryptionKey" xml:space="preserve">
+    <value>Do Not Hard Code Encryption Key</value>
+  </data>
+  <data name="DoNotHardCodeEncryptionKeyDescription" xml:space="preserve">
+    <value>The .Key property or rgbKey parameter of a SymmetricAlgorithm should never be a hardcoded value.</value>
```

Comment:
>The .Key property or rgbKey parameter of a SymmetricAlgorithm should never be a hardcoded value. [](start = 11, length = 96)

SymmetricAlgorithm's .Key property, or a method's rgbKey parameter, should never be a hardcoded value.